src/functions-and-arrays.js

```diff
@@ -29,9 +84,32 @@ const wordsUnique = [
   'bring'
 ];
 
+function uniquifyArray(arr) {
+  let newArr = [];
+  for (let i=0, word=arr.length; i<word ; i++)
+  if(newArr.indexOf(arr[i]) === -1 && arr[i] !== '')
+  newArr.push(arr[i]);
+  return newArr;
+}
+uniquifyArray(words)
+
+
 // Iteration #6: Find elements
 const wordsFind = ['machine', 'subset', 'trouble', 'starting', 'matter', 'eating', 'truth', 'disobedience'];
 
+function doesWordExist(arr, word) {
+  for (let i=0 ; i<arr.length; i++) {
+    if (arr[i] === word) {
+      return true;
+    } else {
+      return false;
+    }
+  }
+}
```

Comment:
Aquí lo ideal sería que solo devolvieses true dentro del for loop, cuando la palabra actualmente iterada (arr[i]) es igual a la palabra buscada (word);
Si no devuelve true, es que la palabra no estaba en el array, y entonces devolvemos false al salir del for loop:
```
function doesWordExist(arr, word) {
  for (let i=0 ; i<arr.length; i++) {
    if (arr[i] === word) {
      return true;
    } 
  }
  return false;
}
```